data/pc/1.20.2/protocol.json

```diff
@@ -28,6 +28,18 @@
     "restBuffer": "native",
     "nbt": "native",
     "optionalNbt": "native",
+    "nbtRoot": [
+      "nbt",
+      {
+        "tagType": "anonymousNbt"
```

Comment:
where is tagType used?

Reply:
`nbt` is a custom type in node-minecraft-protocol, tagType is passed down to the type serializer code for calling prismarine-nbt parsePacketBuffer